src/main/g8/project/BuildHelper.scala

```diff
@@ -6,13 +6,14 @@ import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 import sbtbuildinfo._
 import dotty.tools.sbtplugin.DottyPlugin.autoImport._
 import BuildInfoKeys._
+import scalafix.sbt.ScalafixPlugin.autoImport.scalafixSemanticdb
 import scalafix.sbt.ScalafixPlugin.autoImport._
 
 object BuildHelper {
   val Scala211        = "2.11.12"
   val Scala212        = "2.12.12"
   val Scala213        = "2.13.3"
```

Comment:
```suggestion
  val Scala213        = "2.13.4"
```

:-)

Reply:
We need to keep 2.13.3 for now, Silencer hasn't been published yet for 2.13.4

repo1.maven.org/maven2/com/github/ghik